src/backend/src/prisma/migrations/20230629193452_finance/migration.sql

```diff
@@ -32,6 +32,7 @@ CREATE TABLE "Receipt" (
 -- CreateTable
 CREATE TABLE "Reimbursement_Request" (
     "reimbursementRequestId" TEXT NOT NULL,
+    "identifier" SERIAL NOT NULL,
```

Comment:
I think everything should be on a new migration due to changes on the schema, not edits to old migrations

Reply:
Now I'm thinking about it, what do we meant by the new migration file?

The new migration.sql file in this PR got auto-generated by yarn prisma:migrate command after schema.prisma change.

Reply:
Oh I got what you meant after facing the error: `ERROR: column "identifier" of relation "Reimbursement_Request" already exists` 

I had to just remove this line since I already added "identifier" prop in a new migration file.